react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -95,99 +100,57 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
+                      name="system_file"
+                      label="Custom File Name"
+                      required
+                      className={`${styles['input-custom-size']}`}
+                    />
+                  </span>
+                  <span className={`${styles['hazmapper-custom']}`}>
+                    .hazmapper
+                  </span>
                 </div>
-                {/* Alt solution to render error message bc input group was causing text to not display properly */}
-                {touched.system_file && errors.system_file && (
-                  <div className={`${styles['custom-error-message']}`}>
-                    {errors.system_file}
-                  </div>
-                )}
-              </FormGroup>
-              {/* TODO_REACT: This part will change once the FileBrowser component is added. https://tacc-main.atlassian.net/browse/WG-208*/}
-              <FormGroup className="row align-items-center">
-                <Label className="col-sm-4">Save Location:</Label>
-                <div className="col-sm-8 text-primary">
-                  {userData ? `/${userData.username}` : 'Loading...'}
+                <div className={`${styles['flexContainer-alt']}`}>
+                  <label htmlFor="save-location-label">Save Location:</label>
+                  <span className="text-primary">/{userData?.username}</span>
                 </div>
-              </FormGroup>
-              <FormGroup className="row mb-2">
-                <Label className="col-sm-4 col-form-label pt-0">
-                  Sync Folder:
-                </Label>
-                <div className="col-sm-8">
-                  <div className="form-check">
-                    <Field
-                      type="checkbox"
-                      name="syncFolder"
-                      id="syncFolder"
-                      className="form-check-input"
-                      checked={values.syncFolder}
-                      onChange={handleChange}
-                    />
-                  </div>
-                  <br />
-                  <br />
-                  <Label
-                    className={`${styles['form-check-label']}`}
-                    for="syncFolder"
-                  >
-                    When enabled, files in this folder are automatically synced
-                    into the map periodically.
-                  </Label>
+                <div className={`${styles['flexContainer']}`}>
+                  <label htmlFor="sync-folder-label">Sync Folder:</label>
+                  <span className={`${styles['check-wrapper']}`}>
+                    <FormikCheck name="syncFolder" label="" description="" />
+                  </span>
+                </div>
+                <div className={`${styles['custom-sync-description']}`}>
+                  When enabled, files in this folder are automatically synced
+                  into the map periodically.
                 </div>
```

Comment:
**Yes.** That's an excellent step for now for this PR.

Reply:
I plan to make a new task for this change, and address it early March.

Reply:
Thoughts on the checkbox and description being below the label? See screenshot. This styling/layout is directly in the FormikCheck comp now
![Screenshot 2024-02-26 at 11 53 57 AM](https://github.com/TACC-Cloud/hazmapper/assets/50084480/0dc9afe2-a50b-45a7-a4c2-2f1358b3118e)

Reply:
**Good.** With `FormikCheck` in control of any checkbox, styles can be adjusted[^1] at our leisure[^2] (rather than "fix now so copy-paste does not propagate something different than design").

[^1]: Ideally, it looks like `c-form__field has-type-check` does in [c-form--portal.html](https://tacc.utexas.edu/static/ui/components/detail/c-form--portal.html) ([screenshot](https://github.com/TACC-Cloud/hazmapper/assets/62723358/1b80c7fe-843e-4452-b9cb-37808875600d)), but **zero** rush to achieve that, because `@tacc/core-components` will elt us change it in one spot and propagate.[^2]
[^2]: I'll update [TACC/tup-ui](https://github.com/TACC/tup-ui/) with [TACC-Cloud/hazmapper](https://github.com/TACC-Cloud/hazmapper/) improvements for `@tacc/core-components` release.